README.md

```diff
@@ -32,37 +32,38 @@ High level project goals:
-
-tealdeer reaches these goals. During a (highly non-scientific) test (see
-[#38](https://github.com/dbrgn/tealdeer/issues/38) for details), I tested the
-invocation speed of `tldr <command>` for a few of the existing clients:
-
-| Client | Times (ms) | Avg of 5 (ms) |
-| --- | --- | --- |
-| [Tealdeer](https://github.com/dbrgn/tealdeer/) | `15/11/5/5/11` | `9.4` (100%) |
-| [C client](https://github.com/tldr-pages/tldr-cpp-client) | `11/5/12/11/15` | `10.8` (115%) |
-| [Bash client](https://github.com/pepa65/tldr-bash-client) | `15/19/22/25/24` | `21.0` (223%) |
-| [Go client by k3mist](https://github.com/k3mist/tldr/) | `98/96/100/95/101` | `98.8` (1'051%) |
-| [Python client](https://github.com/lord63/tldr.py) | `152/148/151/158/140` | `149.8` (1'594%) |
-| [NodeJS client](https://github.com/tldr-pages/tldr-node-client) | `169/171/170/170/170` | `170.0` (1'809%) |
-
-tealdeer was the winner here, although the C client and the Bash client are in
-the same speed class. Interpreted languages are clearly much slower to invoke,
-a delay of 170 milliseconds is definitely noticeable and increases friction for
-the user.
-
-These are the clients I tried but failed to compile or run:
-[Haskell client](https://github.com/psibi/tldr-hs),
-[Ruby client](https://github.com/YellowApple/tldrb),
-[Perl client](https://github.com/skaji/perl-tldr),
-[Go client by anoopengineer](https://github.com/anoopengineer/tldr/),
-[PHP client](https://github.com/BrainMaestro/tldr-php).
-
+A tool like `tldr` should be as frictionless as possible to use and show the
+output as fast as possible.
+
+We think that `tealdeer` reaches these goals. We put together a (more or less)
+reproducible benchmark that compiles a handful of clients from source and
+measures the execution times on a cold disk cache. The benchmarking is run in a
+Docker container using sharkdp's [`hyperfine`][hyperfine-gh]
+([Dockerfile][benchmark-dockerfile]).
+
+| Client (500 runs)                     | Mean in ms | Deviation in ms | Comments                                             |
+| :---:                                 | :---:      | :---:           | :---:                                                |
+| `tealdeer`                            | 17.6       | 1.2             |                                                      |
+| [`tldr-c`][c-gh]                      | 36.6       | 2.3             |                                                      |
+| [`tldr-hs`][hs-gh]                    | 22.7       | 2.2             | outputs raw pages, no syntax highlighting of command |
+| [`fast-tldr (Haskell)`][fast-tldr-gh] | 14.8       | 1.3             | outputs raw pages, no syntax highlighting of command |
+| [`tldr-bash`][bash-gh]                | 27.8       | 1.1             |                                                      |
+| [`outfieldr (Zig)`][outfieldr-gh]     | 7.9        | 1.0             | no syntax highlighting of command                    |
```

Comment:
Hi, author of outfieldr here. Just wondering what is meant by "no syntax highlighting of command"?

Reply:
Tealdeer can outtput command names, user variables and the rest of the example code in three completely different styles according to the user config. As far as I could tell, outfieldr only changes the text color of the whole line, so I wanted to note that here (correct me if I am wrong though)

Maybe the wording is a bit misleading, what do you propose as a fairer comment?

---

On another note: I read that zig 0.8 is out now. Should I pin the compiler to 0.8 or do you plan on using the dev version?

Reply:
Outfieldr renders user arguments in bright/bold red, and the rest of the code (including the command name) in red. So if I understand correctly, it's lacking the feature of stylizing the command name in a different style?

---

I will be pinning Outfieldr to stable releases from now on, it'll be easier to maintain that way.

Reply:
Oh sorry, I didnt see that the variables are any different. Good thing I had you look over 🙃

In that case, is it rather a "no config for syntax highlighting" / "no user config"? Or did I just not see this either? 😂

I wouldn't list the fact about the command name as its only a nice addon but not a necessity imo.

Great, Ill make sure to set the compiler image version

Reply:
You're correct about the lack of configuration, there is none at the moment. Might be a good thing to add in the future :thinking:

Reply:
Great, thank you for your cooperation! 👍